src/logion/model/locrequest.model.ts

```diff
@@ -253,36 +253,45 @@ export class LocRequestAggregateRoot {
         if(this.iDenfyVerification && this.iDenfyVerification.status === "PENDING") {
             throw new Error("Cannot submit with ongoing iDenfy verification session");
         }
-        this.status = 'REQUESTED';
+        this.status = 'REVIEW_PENDING';
         this.updateAllItemsStatus("REVIEW_PENDING");
     }
 
     reject(reason: string, rejectedOn: Moment): void {
-        if (this.status != 'REQUESTED') {
+        if (this.status != 'REVIEW_PENDING') {
             throw new Error("Cannot reject already decided request");
```

Comment:
I would use a more generic error message (unexpected status does not always mean "already decided").